README.md

```diff
@@ -165,6 +165,98 @@ We demonstrate the superiority of our MMPose framework in terms of speed and acc
 
 More details about the benchmark are available on [benchmark.md](docs/benchmark.md).
 
+## Inference speed summary
+
+We show the summary of complexity information and inference speed for the major models in MMPose, including FLOPs, parameter counts and inference speeds on both CPU and GPU devices with different batch sizes.
+
+<details open>
+<summary>(click to collapse)</summary>
```

Comment:
The table is too massive to show in the readme. I suggest only keeping a short description and the link to the markdown file here.

Reply:
Good suggestion. I will change it.